README.rst

```diff
@@ -69,3 +69,11 @@ Compatibility
 
 This library should be compatible with py-bcrypt and it will run on Python
 2.6+, 3.2+, and PyPy.
+
+Security
+--------
+
+``bcrypt`` follows the `same security policy as cryptography`_, if you
+identify a vulnerability, we ask you to contact us privately.
+
+.. _`same security policy as cryptography`: https://cryptography.io/en/latest/security/
```

Comment:
We haven't looked at HackerOne for this project or pyca in general,
although I think several of us have used it on other projects (e.g. Django
and Python). Thanks for working on it!

On Fri, Mar 13, 2015 at 10:01 PM, Reed Loden <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/pyca/bcrypt/pull/39#discussion_r26434081>:
>
> > @@ -69,3 +69,11 @@ Compatibility
> >
> >  This library should be compatible with py-bcrypt and it will run on Python
> >  2.6+, 3.2+, and PyPy.
> > +
> > +Security
> > +--------
> > +
> > +``bcrypt`` follows the `same security policy as cryptography`_, if you
> > +identify a vulnerability, we ask you to contact us privately.
> > +
> > +.. _`same security policy as cryptography`: https://cryptography.io/en/latest/security/
>
> Have you thought about using something like https://hackerone.com/ (or
> some of the other platforms/services out there) to help with this? Just
> curious... (disclaimer: speaking as a security engineer here who likes to
> see projects handling security reports better, not as hackerone.com
> employee).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/bcrypt/pull/39/files#r26434081>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084